### PR TITLE
[QC-651] Add class to metadata

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by Barthelemy** [comment]
I preferred updating all references to the MO constructor to make sure that we specify the class everywhere.

**Comment by Barthelemy** [comment]
re-wip it because there is something wrong

**Comment by Barthelemy** [comment]
ok, 1 commit was not pushed

File: Modules/ITS/src/TrendingTaskITSThr.cxx [hunk 1]
```diff
@@ -78,6 +78,7 @@ void TrendingTaskITSThr::storeTrend(repository::DatabaseInterface& qcdb)
   ILOG(Info, Support) << "Storing the trend, entries: " << mTrend->GetEntries() << ENDM;
 
   auto mo = std::make_shared<core::MonitorObject>(mTrend.get(), getName(),
+                                                  "o2::quality_control_modules::its::TrendingTaskITSThr",
```
Comment (knopers8) [inline Modules/ITS/src/TrendingTaskITSThr.cxx:81]: Why not `mConfig.className`?

**Review by knopers8 — APPROVED** [review]
It looks good, but please check that one suggestion.

File: Modules/ITS/src/TrendingTaskITSThr.cxx (hunk 1 above)
Comment (Barthelemy) [reply Modules/ITS/src/TrendingTaskITSThr.cxx:81]: yep

**Pull request merged**